src/Bulk.php

```diff
@@ -323,9 +323,7 @@ protected function _processResponse(Response $response): ResponseSet
                         if (!$data->hasId() && isset($bulkResponseData['_id'])) {
                             $data->setId($bulkResponseData['_id']);
                         }
-                        if (isset($bulkResponseData['_version'])) {
-                            $data->setVersion($bulkResponseData['_version']);
-                        }
+                        $data->setVersionParams($bulkResponseData);
```

Comment:
Now I see where we need this. Is there a better way to do this?

Reply:
I'm not sure there's a better way, I was trying to avoid to copy/paste code

Reply:
I think copy/paste in this case is fine, adding another public method just for avoiding calling each setter doesn't bring so much value.